config/routes.rb

```diff
@@ -33,4 +33,9 @@
   get "/orders/new", to: "orders#new"
   post "/orders", to: "orders#create"
   get "/orders/:id", to: "orders#show"
+
+  get "/register", to: "users#new"
+  resources :users, only: [:create]
```

Comment:
Could we maybe move resources up one line to sort of kick off this block of user routes?

Reply:
It's done!  Nicely done.